README.md

```diff
@@ -1335,6 +1336,35 @@ signIn.on('ready', function (context) {
 });
 ```
 
+### afterError
+
+Triggered after the widget encounters an error during an authentication or authorization flow, and page animations have finished. Returns `error` and `context` objects containing the following properties:
+
+- `error`:
+  - **name** - Name of the error triggered
+  - **message** - Error message
+  - **statusCode** - HTTP status code  (if available)
+  - **xhr** - HTTP response (if available)
+- `context`:
+  - **controller** - Current controller name
+
+```javascript
+signIn.on('afterError', function (error, context) {
```

Comment:
curious why you like to split `controller` to another variable but not keep in `error` object?

Reply:
Primarily to be consistent with all general purpose events, as there will always be a `context` object with information related to the current state. To me, merging `context` with `error` overloads the `error` object.

Reply:
Make sense to split them, however, I think if we want to be consistent in general we should keep the common returned objects in the same positions. 
Just for example, if we have a general callback function that use `context`, we cannot use for both render events and errors event, because in afterRender `context` would be the first parameter, while in afterError event `context` would be the second parameter.

Reply:
Good point - I originally thought the more important param *should* be first, but I do like the idea of consistency between all events.

Updated in a follow-up commit.